### ISSUE TITLE
ADD DEI.md file

### ISSUE THREAD

**Issue by kjaymiller** [BlackPythonDevs/.github#13]
Add a `DEI.md` file per - https://github.blog/2023-06-07-announcing-the-all-in-chaoss-dei-badging-pilot-initiative/

**Comment by oleksis** [comment]
We need fill out the [DEI.md template](https://github.com/AllInOpenSource/ProjectBadging/blob/main/Template.DEI.md#diversity-equity-and-inclusion-project-statement) for BPDevs community.

> At the most basic level, projects should communicate their policies and procedures around four factors:
> * Project access and accessibility
> * Communication transparency
> * Newcomer experience
> * Inclusive Leadership

**Comment by kjaymiller** [comment]
yup it's a process that requires thoughts and has questions we should answer

**Comment by kjaymiller** [comment]
What do you think as a starting point?

---

## Project access and accessibility

- The project is open source under the MIT license and available for contribution.
- Accessibility testing is conducted on the website via the [Playwright-Ally plugin](https://github.com/pamelafox/pytest-axe-playwright-snapshot).

## Communication transparency

- Leadership conversations are done as needed publicly in the [.maintainers repo](https://github.com/orgs/BlackPythonDevs/.maintainers)
- Updates are pushed to Black Python Devs Community channels (Discord, X, Mastodon, LinkedIn)

## Newcomer experience
- All contributors can opt for pairing from repo maintainers based on availability. This can be done via audio/video call or asynchronously via Discord.
- Project repos all contain our license, code of conduct, and contributing documents.

## Inclusive Leadership
- The project is led by the admins of the Black Python Devs community, an online-first community of Python developers who identify as Black. More information about them at https://blackpythondevs.github.io/about
- Leadership is selected based on participation and in discussion with other leaders.
